gspread_pandas/client.py

```diff
@@ -735,6 +748,7 @@ def df_to_sheet(
         start=(1, 1),
         replace=False,
         sheet=None,
+        raw_columns=[],
```

Comment:
yeah, this is kind of confusing since that `raw_columns` of column names actually gets converted into the mapped `raw_columns` variables of ints before getting passed into `self.update_cells` - I think I just couldn't think of a better naming convention at the time

Reply:
Ahh that makes sense. Maybe that one could be called `raw_column_names`?